spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -1347,18 +1322,47 @@ Requests:
 ----
 ====
 
-NOTE: When using the common memory property, you should use an `m` suffix for the value. When using the Kubernetes-specific properties, you should use the Kubernetes `Mi` style suffix.
+You can also control the default values to which to set the `cpu` and `memory` globally, for example:
 
-You can also control the default values to which to set the `cpu` and `memory` requirements for the pods that are created as part of application deployments. You can declare the following as part of the container `env` section in a deployment YAML file:
+For Streams:
 
 ====
 [source,yaml]
 ----
-env:
-- name: SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_KUBERNETES_ACCOUNTS[default]_CPU
-  value: 500m
-- name: SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_KUBERNETES_ACCOUNTS[default]_MEMORY
-  value: 640Mi
+data:
```

Comment:
It seems like we are describing how to override different properties one after the other. It is clear, but then there's the repetition of the same section over and over only to list/describe the new property on each section.

Maybe we could list them all in 1-go and number them inside the YAML, and use that numbering to describe the specific property being overridden? 

Here's an example in the [ref. guide](http://docs.spring.io/spring-cloud-dataflow/docs/1.7.3.RELEASE/reference/htmlsingle/#configuration-security-enabling-https). Notice all the numbers on the right and the legends beneath for the description. It could simplify the docs.